### DISCUSSION TITLE
Has anyone experience issue with visual indicators of columns being sorted despite not being sorted anymore?

### DISCUSSION THREAD

**Discussion by rubenlopezlozoya** [sadmann7/tablecn#755] [General]
I've copied the Data Table into my own project and disabled advanced filtering. Whenever I'm sorting, there's times when the check mark that indicates if a column is sorted desc or asc will appear despite that column not being sorted anymore. This does not happen all the time, and when it happens it's only a visual issue since the sorting works properly and the url params work as intended. Can't figure why this happens as the column.getIsSorted() is also properly set, it's just the check mark not disappearing. 

In the image you can see the url where the table is sorted by the column "query" desc, but the "domain" column still shows a checkmark.

<img width="1310" alt="image" src="https://github.com/user-attachments/assets/9e6ffadf-e224-4c59-918f-eff9d8ae472f">

**Comment by sadmann7** [comment]
the  `useEffect` in `update-task-sheet` was causing infinite rendering.

should be fixed now.
